### PR TITLE
Do not panic while panicking

### PR DESCRIPTION
**Pull request by bkchr**
Fixes: https://github.com/paritytech/stakingops-issues/issues/36

### CODE REVIEW

**Review by NikVolf — APPROVED** [review]

File: primitives/runtime/src/lib.rs [hunk 1]
```diff
@@ -794,8 +794,13 @@ impl SignatureBatching {
 
 impl Drop for SignatureBatching {
 	fn drop(&mut self) {
+		#[cfg(feature = "std")]
```
Comment (NikVolf) [inline primitives/runtime/src/lib.rs:797]: worth adding to `sp-std` as function?

File: primitives/runtime/src/lib.rs [hunk 2]
```diff
@@ -885,4 +890,18 @@ mod tests {
 			);
 		});
 	}
+
+	#[test]
+	#[should_panic(expected = "Hey, I'm an error")]
+	fn batching_does_not_panics_while_thread_is_already_panicking() {
```
Comment (dvdplm) [inline primitives/runtime/src/lib.rs:896]:
```suggestion
	fn batching_does_not_panic_while_thread_is_already_panicking() {
```

**Review by dvdplm — APPROVED** [review]

**Review by cheme — APPROVED** [review]
LGTM :+1:

File: primitives/runtime/src/lib.rs (hunk 1 above)
Comment (bkchr) [reply primitives/runtime/src/lib.rs]: Done

**Comment by bkchr** [comment]
bot merge

**Pull request merged**